### DISCUSSION TITLE
Are lists public or private?

### DISCUSSION THREAD

**Discussion by kristiantorres** [community/community#8811] [Other Feature Feedback, Questions, & Ideas]
Can other users see the lists I create in my profile? If they are private(public), how can we make them public(private)?

Thanks

**Comment by MMK21Hub** [comment]
They're public at the moment, but private lists will come soon
> right now lists are public by default. In the future, we will explore adding the ability to create _private_ lists, but for now you should be good to go!

https://github.com/github/feedback/discussions/8607#discussioncomment-1789256

**Reply by Morphclue** [reply]
They are not fully public. It seems that they are only public for logged-in users.

**Reply by zessu** [reply]
5 yrs later ....

**Comment by Nico7777777** [comment]
Nowadays, they still can't be edited for availability scope, so it takes longer....
